src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -2195,17 +2195,26 @@ case class TableExplode(child: TableIR, column: String) extends TableIR {
 
     val childRowType = child.typ.rowType
 
+    val fieldToExplode = columnType match {
```

Comment:
For consistency, I this this should just be "field", and you should refactor the names "column*" to "field*" in the rest of this method, starting with the parameter "column", which I think should be "fieldName" to distinguish it from this variable. Only our matrix tables have columns.